src/strings/windows1252.c

```diff
@@ -340,25 +340,18 @@ static MVMuint8 windows1251_cp_to_char(MVMint32 codepoint) {
  * buffers, or until a stopper is reached. */
 MVMuint32 MVM_string_windows125X_decodestream(MVMThreadContext *tc, MVMDecodeStream *ds,
                                          const MVMint32 *stopper_chars,
-                                         MVMDecodeStreamSeparators *seps, MVMString *replacement,
-                                         const MVMuint16 *codetable,
-                                         MVMint64 config) {
+                                         MVMDecodeStreamSeparators *seps,
+                                         const MVMuint16 *codetable) {
     MVMint32 count = 0, total = 0;
     MVMint32 bufsize;
     MVMGrapheme32 *buffer = NULL;
     MVMDecodeStreamBytes *cur_bytes = NULL;
     MVMDecodeStreamBytes *last_accept_bytes = ds->bytes_head;
     MVMint32 last_accept_pos, last_was_cr;
     MVMuint32 reached_stopper;
-    MVMStringIndex repl_length;
-    if (replacement) {
-        repl_length = MVM_string_graphs(tc, replacement);
-        if (1 < repl_length) {
-            MVM_exception_throw_adhoc(tc,
-            "Windows-1252 and Windows-1251 decoding with a replacement string with more than one grapheme is not yet implemented.\n"
-            "Provided string had %"PRIu32" graphemes\n", repl_length);
-        }
-    }
```

Comment:
I see this is now supported - but there's a problem still: what if the grapheme in the replacement isn't a valid one under this encoding also?

Reply:
This is just the decoder, so I wouldn't think that would be an issue. Unless you can think of some instances when it could be?

Reply:
Oh, goodness, you're right. Sorry, should probably not do review this close to sleep time. :-)